features/admin.login-flow.ai.v1/hooks/use-generate-ai-login-flow.ts

```diff
@@ -78,6 +78,30 @@ const useGenerateAILoginFlow = (): GenerateLoginFlowFunction => {
                 }
             )
             .catch((error: IdentityAppsApiException) => {
+                if (error?.code === 422) {
```

Comment:
Let's define these error codes in a constants file in a follow up PR.

Reply:
Ack